src/parser.mjs

```diff
@@ -197,6 +220,7 @@ export class ExifParser extends Reader {
 		if (this.options.xmp)  this.parseXmpSegment()  // Additional XML data (in XML)
 		if (this.options.icc)  this.parseIccSegment()  // Image profile
 		if (this.options.iptc) this.parseIptcSegment() // Captions and copyrights
+		if (this.options.flir) this.parseFlirFFFSegment() // Additional data included by FLIR cameras
```

Comment:
I currently require the options to pass `{ flir: true }`. Is this ok, or do you guys prefer to just automatically enable this in our fork?

Reply:
Ok like this